src/__tests__/ReduxRouter-test.js

```diff
@@ -137,6 +137,76 @@ describe('<ReduxRouter>', () => {
     expect(historySpy.callCount).to.equal(1);
   });
 
+  it('should accept React.Components for "RoutingContext" prop of ReduxRouter', () => {
+    const reducer = combineReducers({
+      router: routerStateReducer
+    });
+
+    const history = createHistory();
+    const store = reduxReactRouter({
+      history
+    })(createStore)(reducer);
+
+    store.dispatch(push({ pathname: '/parent/child/123', query: { key: 'value' } }));
+
+    let consoleErrorSpy = sinon.spy(console, 'error');
+
+    renderIntoDocument(
+      <Provider store={store}>
+        <ReduxRouter RoutingContext={RouterContext}>
+          {routes}
+        </ReduxRouter>
+      </Provider>
+    );
+
+    expect(consoleErrorSpy.called).to.be.false;
+
+    renderIntoDocument(
+      <Provider store={store}>
+        <ReduxRouter RoutingContext={(props) => <RouterContext {...props}/>}>
+          {routes}
+        </ReduxRouter>
+      </Provider>
+    );
+
+    expect(consoleErrorSpy.called).to.be.false;
+
+    console.error.restore(); // eslint-disable-line no-console
+
+    class CustomRouterContext extends React.Component {
+      render(props) {
+        return <RouterContext {...props}/>;
+      }
+    }
+
+    consoleErrorSpy = sinon.spy(console, 'error');
+
+    expect(() => renderIntoDocument(
```

Comment:
You can put this function `() => renderIntoDocument` into a separate variable. This will improve readability of the `expect` statement. Same with the error message String below.